starport/services/chain/command.go

```diff
@@ -0,0 +1,101 @@
+package chain
```

Comment:
Any reason why we've moved this from `init.go` to here? We call/construct binary commands almost everywhere in the `chain` pkg and it is not only limited to `command.go`.

Reply:
Because the commands are not related specifically to the initialization phase